### DISCUSSION TITLE
A simple way to send data via zigbee?

### DISCUSSION THREAD

**Discussion by Ismola** [espressif/arduino-esp32#9368] [Q&A] [answered]
I have looked at the **zigbee example** to turn a light bulb on and off, but it seems very complex to me for the simple idea it is.

Is there a way to make it simpler?

**Comment by VojtechBartoska** [comment, marked as answer]
Hello,

there is also a Readme explaining it, it should help.

https://github.com/espressif/arduino-esp32/tree/master/libraries/ESP32/examples/Zigbee/Zigbee_Light_Switch
